From 18a990d4b6677ba7e3e7b0d4a2980864da4826a5 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Tue, 16 Aug 2016 20:23:59 +0500
Subject: [PATCH] layers: obey CXX_FLAGS

The CMakeLists earlier simply dropped all the CXX_FLAGS
set throught he build system. This limits any chances
of providing custom flags such as linking directories
etc.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 layers/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/layers/CMakeLists.txt b/layers/CMakeLists.txt
index b384803..c67f1df 100644
--- a/layers/CMakeLists.txt
+++ b/layers/CMakeLists.txt
@@ -99,8 +99,7 @@ if (WIN32)
     set (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG} -D_CRT_SECURE_NO_WARNINGS /bigobj")
     set (CMAKE_C_FLAGS_DEBUG     "${CMAKE_C_FLAGS_DEBUG} -D_CRT_SECURE_NO_WARNINGS /bigobj")
 else()
-    set (CMAKE_CXX_FLAGS "-std=c++11")
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wpointer-arith")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wpointer-arith")
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wpointer-arith")
 endif()
 
-- 
1.9.1

